legal-api/src/legal_api/core/filing.py

```diff
@@ -531,7 +531,9 @@ def get_document_list(business,  # pylint: disable=too-many-locals disable=too-m
             Filing.FilingTypes.AGMEXTENSION.value,
             Filing.FilingTypes.AGMLOCATIONCHANGE.value,
         ]
-        if filing.status in (Filing.Status.PAID, Filing.Status.WITHDRAWN) and \
+        if (filing.status in (Filing.Status.PAID, Filing.Status.WITHDRAWN) or
```

Comment:
is `WITHDRAWN` actually a filing status for the filing being withdrawn?  I don't think I see this status for latest withdrawn filings in dev.  Or is that